hazelcast-client-legacy/src/test/java/com/hazelcast/client/replicatedmap/ClientReplicatedMapListenerTest.java

```diff
@@ -115,6 +117,50 @@ public void run() throws Exception {
         });
     }
 
+
+    @Test
+    public void testListenToKeyForEntryAdded() throws Exception {
+        HazelcastInstance instance = factory.newHazelcastInstance();
+        HazelcastInstance client = factory.newHazelcastClient();
+        String mapName = randomMapName();
+        ReplicatedMap<Object, Object> replicatedMap = client.getReplicatedMap(mapName);
```

Comment:
Can't this be pulled out into the setup of the test since each test has exactly the same beginning.  So everything above this comment.